supervision/detection/utils.py

```diff
@@ -274,6 +275,81 @@ def box_non_max_suppression(
     return keep[sort_index.argsort()]
 
 
+def box_non_max_merge(
```

Comment:
This one seems like a helper function that does not need to be exposed in the API.

Reply:
renamed to `_box_non_max_merge_all`, removed from __init__.py.

Note: you'll see `box_non_max_merge` which is the prior batch function. Now it's the main one.